cg/meta/archive/ddn_dataflow.py

```diff
@@ -188,23 +192,29 @@ def auth_header(self) -> Dict[str, str]:
             self._refresh_auth_token()
         return {"Authorization": f"Bearer {self.auth_token}"}
 
-    def archive_folders(self, sources_and_destinations: List[MiriaFile]) -> int:
+    def archive_folders(self, files_and_samples: List[FileAndSample]) -> int:
```

Comment:
Out of scope, but I think it is less confusing to return the actual `TransferJob` model from these methods rather than an int

Reply:
Yes, I have an intention to create a PR where I create response class extending request.Response or something similar. In general, I think we should expect a request.Response object.

Reply:
Or a data class modelling a successful response.